AndroidBound/src/main/java/solutions/alterego/androidbound/ViewBinder.java

```diff
@@ -75,7 +73,7 @@
 
     private IImageLoader mImageLoader = IImageLoader.nullImageLoader;
 
-    public ViewBinder(Context ctx, IAndroidLogger logger) {
+    public ViewBinder(Context ctx, ILogger logger) {
```

Comment:
well, if you set it later, you will lose the logging of the setup since everything is started from the constructor. but I added a separate constructor without the `ILogger` now so it can be used without it (the setter was already there).